### PR TITLE
WIP Hass.io panel remake, RFC

### PR DESCRIPTION
**Pull request by ciotlosm**
## Hass.io panel remake - Request for Change

- [ ] all "good" buttons on right side (`float: right` seems to kill div hight here....), all "evil" buttons red and on the left side
- [ ] use the grey from /config/dashboard for our card-groups (it does not seem to be --primary-text-color <-- looks black when I use it..)
- [ ] new layout for /advanced: maybe all cards on the left (one card group for each) and the log in the right side with a refresh button
- [ ] new layout for /addons, no idea how...
- [ ] talk about error boxes, where are they usefull, are they usefull, where to place them
- [ ] updating data after some APIs are called
- [ ] vertical center card content
- [ ] make the create snapshot button get a spinner (like in old panel on some buttons)
- [ ] check if we get errors when pressing a reboot/update button -> pvizeli should send us an HTTP 200 OK before rebooting
- [ ] scrolling bug (on mobile)
- [ ] scroll back to top when change tab on mobile
- [ ] ask pvizeli to provide all data with 1 API call
- [ ] reduce number of APIs called at the same time - maybe just call the APIs when view requries it
- [ ] align top panels when multiple columns and panel size is different
- [ ] capitalize only first letter (snapshot details name, addon details name, item-info)
- [ ] handle errors for supervisor 'It is already a xxx process running' like `snapshot/restore`
- [ ] snapshot details to use backdrop

## For PR
- [ ] cc: @pvizeli @frenck @cogneato
- [ ] provide a .zip `with home-assistant-polymer` and info how to setup/ update for testers
- [ ] provide screenshots, also 1 from mobile

screen shot 2018-02-05 at 21 18 11
screen shot 2018-02-05 at 21 18 22
screen shot 2018-02-05 at 21 18 29
screen shot 2018-02-05 at 21 19 13
screen shot 2018-02-05 at 21 19 33

### CODE REVIEW

File: hassio/hassio-supervisor/hassio-supervisor.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
```
Comment (balloob) [inline hassio/hassio-supervisor/hassio-supervisor.html]: Why did you move this file from `supervisor` to `hassio-supervisor`?

**Review by balloob — CHANGES_REQUESTED** [review]
Because you moved almost every file, it makes it very difficult to even review this PR. What is new? What is old?

If you move files with git, use `git mv <old> <new>`, that way Git knows it's the same file and keeps the history. This needs to be addressed or else reviewers will be unable to see what has changed in this PR.

Also, don't do code drops like this. One change per PR! It's the only way that people will even take the time to do a review. I can't compare 2500 lines changes and see what is going on. If we can't see what is going on, we can't judge if this is an improvement or not.

**Comment by c727** [comment]
yes, maybe some things are a bit thoughtless of us. we will split it in smaller changes :heart:

**Comment by ciotlosm** [comment]
I guess the rebase required after the iframe refactor for hassio caused us some problem as well. I'll close this PR and as @c727 said, we'll look at making smaller increments.

**Comment by balloob** [comment]
Sorry if I sounded a bit harsh, was not my intention. You are doing a great job! 🎉
